frontend/Makefile

```diff
@@ -0,0 +1,22 @@
+all: check-coverage lint build
+check-coverage:  test
```

Comment:
nit: two spaces between colon and `test`

Reply:
Done.